lib/sentry.ex

```diff
@@ -124,16 +124,18 @@ defmodule Sentry do
 
       config :sentry,
         before_send: {MyModule, :before_send},
-        after_send_event: {MyModule, :after_send}
+        after_send_event: {MyModule, :after_send_event}
```

Comment:
Can we instead change the function name in the code example to be `:after_send`?